tests/Integration/ApiPlatform/ApiTestCase.php

```diff
@@ -40,4 +59,38 @@ protected static function createClient(array $kernelOptions = [], array $default
 
         return parent::createClient($kernelOptions, $defaultOptions);
     }
+
+    protected function getBearerToken(array $scopes = []): string
+    {
+        $client = static::createClient();
```

Comment:
```suggestion
        if (null === self::$clientSecret) {
           self::createApiAccess($scopes);
        }
        $client = static::createClient();
```

Maybe this can be convenient, this way no need to create the APIAccess in the setUp in most cases this should be enough, what do you think?